atproto/identity/redisdir/redis_directory.go

```diff
@@ -317,11 +365,19 @@ func (d *RedisDirectory) Purge(ctx context.Context, a syntax.AtIdentifier) error
 	handle, err := a.AsHandle()
 	if nil == err { // if not an error, is a handle
```

Comment:
why is this backwards everywhere? is this a rust thing?

Reply:
I think Bryan does this intentionally to make it clear it's a `==` and not a `err != nil`